webui/react/src/constants/states.ts

```diff
@@ -121,6 +121,7 @@ export const resourceStateToLabel: { [key in ResourceState]: string } = {
   [ResourceState.Terminated]: 'Terminated',
   [ResourceState.Pulling]: 'Pulling',
   [ResourceState.Starting]: 'Starting',
+  [ResourceState.Warm]: 'Warm',
```

Comment:
I assume "Warm" is a new term we're going to use to describe these available empty slots to distinguish them from "theoretically" total available ones

Reply:
Yes, warm slots are the available slots among connected agents.

Reply:
Yes, warm slots are the available slots among connected agents.